scripts/provisioning/initcmd.py

```diff
@@ -57,6 +57,9 @@ def process(self):
                                 }
       LdapAccountAction(self.ldap_user, self.ldap_passwd).create_account(bgdelete_acc_input_params_dict)
     except Exception as e:
-      self.logger.error(f'Failed to create backgrounddelete service account, error: {e}')
-      raise e
+      if "Already exists" not in str(e):
```

Comment:
I guess we can have more clean code by adding except block for ldap.ALREADY_EXISTS